### PR TITLE
Add pdfminer to tox.ini and dev-requirements.txt to enable pdf test

### PR DESCRIPTION
**Pull request by cjmayo**
Inspired by https://github.com/linkchecker/linkchecker/issues/317#issuecomment-544536320 and #323.

Let's see how the tests go.

### CODE REVIEW

**Comment by cjmayo** [comment]
Failure without changing PdfParser to use get_raw_content():
```
=================================== FAILURES ===================================
______________________________ TestFile.test_pdf _______________________________
tests/__init__.py:95: in newfunc
    return func(*args, **kwargs)
tests/checker/test_file.py:77: in test_pdf
    self.file_test("file.pdf", confargs=confargs)
tests/checker/__init__.py:238: in file_test
    self.fail_unicode(msg)
tests/checker/__init__.py:248: in fail_unicode
    self.fail(msg)
E   AssertionError: b'tests/checker/data/file.pdf
E   --- 
E   
E   +++ 
E   
E   @@ -1,13 +0,0 @@
E   
E   -http://www.example.com/link1
E   -cache key http://www.example.com/link1
E   -real url http://www.example.com/link1
E   -error
E   -url http://www.example.com/link2
E   -cache key http://www.example.com/link2
E   -real url http://www.example.com/link2
E   -error
E   -url url file:///home/travis/build/linkchecker/linkchecker/tests/checker/data/file.pdf
E   -cache key file:///home/travis/build/linkchecker/linkchecker/tests/checker/data/file.pdf
E   -real url file:///home/travis/build/linkchecker/linkchecker/tests/checker/data/file.pdf
E   -name tests/checker/data/file.pdf
E   -valid'
```

File: tox.ini [hunk 1]
```diff
@@ -7,6 +7,7 @@ deps =
     -rrequirements.txt
     pyftpdlib
     parameterized
+    pdfminer < 20191010
```
Comment (mgedmin) [inline tox.ini]: What happened on October 10 that broke our code?

Comment (cjmayo) [reply tox.ini]: They went Python 3 only (sounds like a great idea....).

Comment (mgedmin) [reply tox.ini]: Thank you for clarifying!

**Pull request merged**